setuptools/config.py

```diff
@@ -255,6 +256,43 @@ def _parse_list(cls, value, separator=','):
 
         return [chunk.strip() for chunk in value if chunk.strip()]
 
+    @classmethod
+    def _parse_list_glob(cls, value, separator=','):
+        """Equivalent to _parse_list() but expands any glob patterns using glob().
+
+        However, unlike direct calls to glob(), the resolved results remain relative paths.
+
+        :param value:
+        :param separator: List items separator character.
+        :rtype: list
+        """
+        glob_characters = ('*', '?', '[', ']', '{', '}')
+        get_relpath = lambda value: os.path.relpath(value, os.getcwd())
+        values = cls._parse_list(value, separator=separator)
+        expanded_values = []
+        for value in values:
+            trimmed_value = value.strip()
+
+            # Has globby characters?
+            if any(char in value for char in glob_characters):
+                value = os.path.abspath(value)
+
+                # check if this path has globby characters but does in fact exist:
```

Comment:
Thanks for thinking of this case. I also agree it's an edge case and unlikely to be depended on, especially since `*` matches `*` and `?` matches `?`. The only situation where this would be a problem is if there were _also_ adjacent data files that were not meant to be matched by the glob but were after this change. That seems remote enough a possibility that I'd prefer to defer this special handling until such a time that it's needed.

Reply:
Fair enough. I'll take it out.